pombola/south_africa/templatetags/za_people_display.py

```diff
@@ -0,0 +1,29 @@
+from django import template
+
+register = template.Library()
+
+NO_PLACE_ORGS = ('parliament', 'national-assembly', )
+MEMBER_ORGS = ('parliament', 'national-assembly', )
+
+
+@register.assignment_tag()
+def should_display_place(organisation):
+    should_display = True
+
+    if organisation.slug in NO_PLACE_ORGS:
+        should_display = False
+
+    return should_display
+
+
+@register.assignment_tag()
+def should_display_position(organisation, position):
```

Comment:
I think the second parameter here is a position title, not a position, so renaming the variable accordingly would be good.